libs/payments/ui/src/lib/client/hooks/useCallbackOnce.tsx

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { useCallback, useRef } from 'react';
+
+export function useCallbackOnce(cb: () => void, deps: any[]) {
```

Comment:
This feels like something to make sure @xlisachan is also aware of - it's a useful new hook!